.github/workflows/ci.yml

```diff
@@ -92,9 +96,24 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: 'pip'
           cache-dependency-path: setup.cfg
+      - run: echo "CRDS_SERVER_URL=https://jwst-crds.stsci.edu" >> $GITHUB_ENV
+        if: contains( matrix.toxenv, '-jwst')
+      - run: echo "CRDS_SERVER_URL=https://roman-crds.stsci.edu" >> $GITHUB_ENV
+        if: contains(matrix.toxenv, '-roman')
+      - run: pip install crds
+        if: contains(matrix.toxenv, '-jwst') || contains(matrix.toxenv, '-roman')
+      - run: echo "pmap=$(crds list --operational-context)" >> $GITHUB_OUTPUT
+        id: crds-context
+      - uses: actions/cache@v3
+        with:
+          path: ${{ env.CRDS_PATH }}
+          key: crds-${{ steps.crds-context.outputs.pmap }}
+        if: contains(matrix.toxenv, '-jwst') || contains(matrix.toxenv, '-roman')
+      - run: crds sync --contexts ${{ steps.crds-context.outputs.pmap }}
+        if: contains(matrix.toxenv, '-jwst') #|| contains(matrix.toxenv, '-roman')
```

Comment:
this line should be uncommented when Roman's CRDS server becomes publicly accessible

Reply:
No that is not expected; it looks like they are attempting to reach the CRDS server. I'll look into if it's possible to run the non-CRDS tests.